test/NuGet.Core.FuncTests/NuGet.Protocol.FuncTest/HttpRetryHandlerTests.cs

```diff
@@ -176,6 +178,108 @@ public async Task HttpRetryHandler_HandlesNameResolutionFailure()
 #endif
         }
 
+        [Fact]
+        public async Task HttpRetryHandler_429WithRetryAfterHeader_UsesHeaderDelay()
```

Comment:
The name seems to suggest the delay is being used, but the code is actually asserting the delay not being used.

Reply:
The "success condition" is "uses header delay". I can rename it to `UsesRetryAfter`, but that's exactly what I meant my "header delay". It's not the programmatic delay specified in the `HttpSourceRequest` object, or the "enhanced retry" environment variable.  Anyway, I guess `UsesRetryAfter` is clearer.

Reply:
I see. Yeah I misunderstood the test because there's multiple ways of configuring it.